feature-libs/product/configurators/common/src/components/config-attribute/config-attribute-header/config-attribute-header.component.html

```diff
@@ -3,7 +3,7 @@
   class="cx-config-attribute-label"
   [class.cx-config-attribute-label-required-icon]="attribute.required"
   [class.cx-config-attribute-label-required-error]="
-    showRequiredMessage() | async
+    showRequiredMessageForDomainAttribute() | async
```

Comment:
If you're not passing a parameter to the method, we can consider converting `showRequiredMessageForDomainAttribute()` to a property.
What do you think? It's just more aligned with the rest of the code-base.

Reply:
After taking a deeper look: No, it seems to be better a leave it as a method, because if refers to @input attributes. So one would need to initialize it in ngOnInit which adds even more complexity

Reply:
Done (see my comment)

Reply:
Hmm, as it depends on the `Input()`s, let's implement `OnInit` then, to make sure sure that we use Inputs when ready.
You can just declare properties e.g. `showRequiredMessageForDomainAttribute$: Observable<?>;` and initialize them in the `ngOnInit` lifecyle hook.